Applications/Spire/Source/KeyBindings/KeyBindingItemDelegate.cpp

```diff
@@ -59,8 +54,15 @@ void KeyBindingItemDelegate::on_editing_finished() {
 bool KeyBindingItemDelegate::eventFilter(QObject* watched, QEvent* event) {
   if(event->type() == QEvent::KeyPress) {
     auto e = static_cast<QKeyEvent*>(event);
-    if(e->key() == Qt::Key_Tab || e->key() == Qt::Key_Backtab) {
-      m_key_signal(static_cast<Qt::Key>(e->key()));
+    if(e->key() == Qt::Key_Tab) {
```

Comment:
Keep `if(... || ...)`, then branch on `if(e->key() == Qt::Key_Tab)` for `closeEditor`.